.automation.conf/tempest/tempest-preprod.overrides.conf

```diff
@@ -0,0 +1,29 @@
+# BEGIN ANSIBLE MANAGED BLOCK
+# WARNING: This file was generated using shakespeare.
+# See: https://github.com/stackhpc/shakespeare
+# END ANSIBLE MANAGED BLOCK
+[DEFAULT]
+openstack_client_http_timeout = 300
+
+[auth]
+compute_quotas = cores:-1,ram:-1
+
+[identity]
+v3_endpoint_type = publicURL
+
+[compute]
+min_microversion = 2.1
+max_microversion = 2.72
+
+[service-clients]
+http_timeout = 600
+[service_available]
+cinder = false
+
+[compute-feature-enabled]
+resize = true
+live_migration = true
+
+[network]
+public_network_id = 13047566-bbf7-4e1d-941b-58e7b571ca8d
```

Comment:
I guess we need to some change this between prod and preprod, eventually.

Reply:
yeah, the plan was to use the ENVIRONMENT environment variable to switch to using a different file